frappe/public/js/frappe/utils/help.js

```diff
@@ -17,7 +17,7 @@ frappe.help.show = function(doctype) {
 
 frappe.help.show_video = function(youtube_id, title) {
 	if (frappe.utils.is_url(youtube_id)) {
-		const expression = '(?:youtube.com/(?:[^/]+/.+/|(?:v|e(?:mbed)?)/|.*[?&]v=)|youtu.be/)([^\"&?/s]{11})';
+		const expression = '(?:youtube.com/(?:[^/]+/.+/|(?:v|e(?:mbed)?)/|.*[?&]v=)|youtu.be/)([^\"&?\\s]{11})';
```

Comment:
```suggestion
		const expression = '(?:youtube.com/(?:[^/]+/.+/|(?:v|e(?:mbed)?)/|.*[?&]v=)|youtu.be/)([^\"&?\s]{11})';
```

extra slash escape slash and tries to match '\' and 's'. Please confirm

Reply:
@surajshetty3416 the `\` is necessary for considering it as whitespace. If I just write `\s` it is considering it as `s` and `\\s` is getting considered as `\s`

Check:

![regex](https://user-images.githubusercontent.com/24353136/88624450-9d554500-d0c4-11ea-9a8b-11a658592d7e.gif)

The sider issue is for a useless escape character `"` in the regex and not the part in question.

Reply:
Oh... got it... `expression` is a string literal instead of regex literal which is why the reverse is applicable (i.e., regex token needs to be escaped)

Reply:
Yup